crawl-ref/source/mon-act.cc

```diff
@@ -2222,6 +2222,63 @@ static void _torpor_snail_slow(monster* mons)
     }
 }
 
+/**
+ *  Make this monster use the magic of a holy lantern. (Vaud).
+ *
+ *  @param mons       The monster holding the lantern.
+ */
+static void _vaud_lantern_burn(monster *mons)
+{
+    //hurt player
+    //check in range and in LOS
+    if (cell_see_cell(you.pos(), mons->pos(), LOS_SOLID_SEE)
+        && grid_distance(mons->pos(), you.pos()) < 4)
+    {
+        //check divine protection
+        if (!is_good_god(you.religion)
```

Comment:
We've moved away from monsters that are unthreatening to players with good gods - it's thematic, but it just doesn't create exciting gameplay. (See e.g. angels no longer being auto-pacified.)

Is this exception necessary here? If we want this 'holy lantern' effect, would it be better to theme it as scouring the undeserving (aka everyone) alike?